content/push/device.textile

```diff
@@ -15,109 +16,114 @@ redirect_from:
 
 To push notifications with Ably, you must first configure the device using its platform-specific notification service (APNs for iOS or FCM for Android) and integrate it with Ably's infrastructure. You can then activate the push notifications service, either directly or via a server.
 
-h3(#configure). Configure devices
+h2(#configure). Configure devices
 
 Configuration is the first step in setting up push notifications with Ably. This step requires setting up the necessary infrastructure on the device's operating system or platform, such as iOS with "APNs":https://developer.apple.com/notifications/ or Android with "FCM":https://firebase.google.com/docs/cloud-messaging, as well as on the Ably platform.
 
 <a href="@content/diagrams/push-process-2.png" target="_blank">
   <img src="@content/diagrams/push-process-2.png" style="width: 100%" alt="Push Notifications in Ably">
 </a>
 
-h3. Configure an Android device
+h3. Configure FCM for Android devices
 
 * Go to the "Firebase Console.":https://firebase.google.com/
 * Click *add project* and follow the steps to *create and manage service account keys*.
-* Download your Service account key file containing your Privacy Enhanced Mail (PEM) cert and PEM private key.
+* Download your Service account JSON file.
 * In your Ably "dashboard":https://ably.com/accounts, navigate to the *notifications tab* under your app settings.
 * Go to *push notifications setup*, click *configure push*.
-* Add your *PEM cert* and *PEM private key*.
+* Add your *service account JSON file* to the *setting up Firebase cloud messaging* section.
 
-h3. Configure an iOS device
+h3. Configure APNs for iOS devices
 
 * Go to the "Apple Developer Program.":https://developer.apple.com/programs/
 * Use the Apple Developer portal to create a *push notification* service certificate for your app.
 * Export the certificate as a *.p12* file.
 * In your Ably "dashboard":https://ably.com/accounts, navigate to the *notifications tab* under your app settings.
 * Go to *push notifications setup*, click *configure push*.
-* Add your *.p12* file.
+* Add your *.p12* file *setting up Apple push notification service* section.
 
-h4. Add Ably Flutter to your iOS project
+h3. Install Ably and set up your SDK
 
-Include Ably Flutter in your project by adding it to your @pubspec.yaml@ or running:
+To install Ably to your app, first "install the Ably SDK":/docs/getting-started/quickstart. Use your preferred package manager or method, such as CDN, NPM, or other systems relevant to your development environment.
 
-```[flutter]
-flutter pub add ably_flutter
+```[java]
+// The Ably SDK is available on GitHub and hosted on Maven Central.
+// To get started with your project, add the SDK into your build.gradle dependencies section and a reference to Maven in the repositories section if it isn't included by default.
```

Comment:
I'd suggest these should be shorter comments for all languages rather than copying the quickstart text personally, e.g.: 

> Add the Ably SDK into your build.gradle dependencies and ensure there's a reference to Maven in the repositories section: